build.gradle

```diff
@@ -19,7 +19,7 @@ project.ext {
   // The directory that will be used to reference dependent libraries.
   // Modify as needed. Defaulted to users home directory.
   libraryDirectory = "${System.getProperty("user.home")}/libs/"
-  imaAndroidSdkZipFileName = "ima-android-v3-3.1.3.zip"
+  imaAndroidSdkZipFileName = "ima-android-v3-3.7.4.zip"
```

Comment:
We should remove the logic to download the IMA ZIP file and just use Jcenter, the same way we did for the SDK